kolibri/plugins/user_profile/assets/src/machines/changeFacilityMachine.js

```diff
@@ -157,13 +160,12 @@ export const changeFacilityMachine = createMachine({
       on: {
         NEW: 'createAccount',
         CONTINUE: 'isAdmin',
-        BACK: 'changeFacility',
       },
     },
     isAdmin: {
       always: [
         {
-          cond: context => context.role === 'superadmin',
+          cond: context => context.role === 'superuser',
```

Comment:
actually, I thought it was something that had decided to be changed in 0.16. I just saw that I was receiving `superuser` instead of  `superadmin` as I expected and I changed it. I didn't investigate why, just supposed it was an intended change. If it is not, we'll have to find out where in the backend this has been changed.

Reply:
@rtibbles - is this something you can answer? 😄